### PR TITLE
gstreamer-runtime 1.22.6

### PR DESCRIPTION
**Pull request by stonerl**
After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by p-linnane** [comment]
We already have a `gstreamrer` formula in Homebrew-core. Why do we need to add a cask?

**Comment by stonerl** [comment]
This one is different from the formula, isn't it? It installs directly to `/Library/Frameworks/GStreamer.framework. It is used by different projects, e.g., [CXPatcher](https://github.com/italomandara/CXPatcher), [wineskin](https://github.com/Gcenx/WineskinServer).

Furthermore, with this version, mkv files, for example, are not treated as other documents in Finder anymore, but rather video files.

**Comment by stonerl** [comment]
Should I rename the cask to, e.g., `gstreamer-framework` to better differentiate it from the formula?

**Comment by SMillerDev** [comment]
Why not just extend the formula to install the framework? This seems like it doesn't add any value that the formula couldn't

**Comment by SMillerDev** [comment]
Both casks and formulae only install the software for the architecture brew is run with.

**Review by miccal — DISMISSED** [review]

**Review by miccal — DISMISSED** [review]

**Review by miccal — DISMISSED** [review]

File: Casks/g/gstreamer-runtime.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  name "GStreamer runtime package"
+  desc "Open Source Multimedia Framework"
+  homepage "https://gstreamer.freedesktop.org/"
+
+  livecheck do
+    url "https://gstreamer.freedesktop.org/download/"
+    regex(/gstreamer[._-]1\.0[._-]v?(\d+(?:\.\d+)+)[._-]universal\.pkg/i)
+  end
+
+  conflicts_with formula: "gstreamer"
```
Comment (razvanazamfirei) [inline Casks/g/gstreamer-runtime.rb:15]: Does this conflict with `gstreamer-development`?

Comment (razvanazamfirei) [reply Casks/g/gstreamer-runtime.rb]: Sounds good! Thank you.

**Review by razvanazamfirei — APPROVED** [review]

**Pull request merged**